src/main/java/subway/domain/ShortestPath.java

```diff
@@ -0,0 +1,38 @@
+package subway.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+public class ShortestPath {
+    private static final double BASE_DISTANCE = 10;
+    private static final double ADDITIONAL_FARE_DISTANCE = 50;
+    private static final int FIRST_ADDITIONAL_DISTANCE_INTERVAL = 5;
+    private static final int SECOND_ADDITIONAL_DISTANCE_INTERVAL = 8;
+    private static final int ADDITIONAL_FARE = 100;
+    private static final int BASIC_FARE = 1250;
```

Comment:
```suggestion
    private static final int BASIC_FARE = 1_250;
```
저는 숫자에 대해서 ```_```를 이용해 가독성을 나타내는 편인데 어떨까요? 😉

Reply:
가독성이 훨씬 좋아지네요! 적용했습니다 ㅎㅎ